### ISSUE TITLE
Reduced moderation mode

### ISSUE THREAD

**Issue by emagaliff** [inspircd/inspircd#1132]
[Charbydis](https://github.com/charybdis-ircd/charybdis) (and possibly other IRCds) has a "reduced moderation" mode (+z) where `When +z is set, the effects of +b, +q, and +m are relaxed. For each message, if that message would normally be blocked by one of these modes, it is instead sent to all the users who are currently set +o (channel operator).`

It's intended use is for moderated debates, but it is also very helpful for any room that deals with controversial discussions or heavy topics. A similar feature / module on InspIRCd would be fantastic and make moderated certain types of channels much easier.

**Comment by SaberUK** [comment]
:+1:

**Comment by Heufneutje** [comment]
:+1:

**Comment by del6597** [comment]
:+1:

**Comment by GLolol** [comment]
This looks like a duplicate of #226. 

There's a [module doing just this in InspIRCd 2.1](https://github.com/inspircd/inspircd/blob/obsolete/insp21/src/modules/m_opmoderated.cpp), and a PR to backport that to 2.0 in extras: inspircd/inspircd-extras#64. It uses channel mode `+U` instead of `+z`, since the latter is already used by `m_sslmodes`.
